lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -184,12 +184,12 @@ uncheck_checkbox:
     op: click
 check_link_and_text:
   element:
-    selector: &text_link
-      xpath: //a[contains(text(),'<text>') and @href='<link_url>']
+    selector: 
+      xpath: //a[contains(text(),'<text>') and contains(@href,'<link_url>')]
 click_link_with_text:
   element:
     selector:
-      <<: *text_link
+      xpath: //a[contains(text(),'<text>') and contains(@href,'<link_url>')]
```

Comment:
split these 2 rules is already agreement with @yapei . This is for calling `check_link_and_text` individually and repeating by another rule, otherwise the params `text` and `link_url` cannot be directly called multiple times in ONE rule.